src/yunohost/app.py

```diff
@@ -640,36 +633,13 @@ def app_upgrade(app=[], url=None, file=None, force=False, no_safety_backup=False
         # Execute the app upgrade script
         upgrade_failed = True
         try:
```

Comment:
there is a `try`, `finally`, but no except, is it ok to do that?

Reply:
try finally without except is valid in python. It allows to execute the code in the finally even if an exception is raised